docs/en/observability/aws-serverless-troubleshooting.asciidoc

```diff
@@ -5,6 +5,11 @@
 <titleabbrev>Troubleshooting</titleabbrev>
 ++++
 
+== New ID generation logic starting from v1.6.0

```

Comment:
let's add an anchor to this paragraph, so that we can link it directly at https://github.com/elastic/elastic-serverless-forwarder/pull/214/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R8